assets/components/odf-lvm/topolvm-controller_deployment.yaml

```diff
@@ -54,8 +54,8 @@ spec:
           timeoutSeconds: 1
         resources:
           requests:
-            cpu: 250m
-            memory: 250Mi
+            cpu: 2m
+            memory: 31Mi
```

Comment:
Why are we so "precise" in resource requests? Should we not give it a bit of an extra by rounding the numbers up?

Reply:
AFAICT the numbers _are_ already rounded up. While safety margins make sense, let’s also consider these are minimum _requests_ used for scheduling and containers can burst higher (as no limit is set) if resources are available.

We may consider rounding to numbers that “feel less precise” (31Mi —> 32MB, 9Mi —> 10Mi), but then let’s work with those for now.

We also want to run those by the ODF team, of course.

Reply:
I had a conversation with @jerpeter1 about this and it looks like we're all on the same page. 
Let's try working with these values for now and follow-up with the ODF team separately.